test/CMakeLists.txt

```diff
@@ -44,6 +44,45 @@ option( MIOPEN_TEST_MLIR "Add tests for MLIR -- EXPERIMENTAL" ${MIOPEN_USE_MLIR}
 option( WORKAROUND_ISSUE_898 "" ON)
 option( WORKAROUND_ISSUE_936 "" ON)
 
+# Detect GPU type for testing
+if(NOT (MIOPEN_TEST_VEGA OR MIOPEN_TEST_GFX908))
+    find_program(ROCMINFO
+        NAMES rocminfo
+        PATHS
+            ${ROCM_ROOT}
+            /opt/rocm
+        PATH_SUFFIXES
+            /bin
+        NO_DEFAULT_PATH
```

Comment:
Remove the `NO_DEFAULT_PATH` as this prevents the user specifying dependencies with `CMAKE_PREFIX_PATH`.

Reply:
Done.